dis_snek/models/context.py

```diff
@@ -252,6 +252,11 @@ def from_dict(cls, data: Dict, client: "Snake") -> "ComponentContext":
         new_cls.component_type = data["data"]["component_type"]
         new_cls.message = client.cache.place_message_data(data["message"])
 
+        if "values" in data["data"]:
+            new_cls.values = data["data"]["values"]
+        else:
+            new_cls.values = []
```

Comment:
You could have just used `data['data'].get("values", [])`

Reply:
😅 
Should I try and revert it or just add another commit?

Reply:
Just add another commit